recipe/meta.yaml

```diff
@@ -55,7 +55,7 @@ test:
   commands:
     - pip check
     - coverage run --source anyio --branch -m pytest --tb=long --color=yes -k="not (bind_link_local or block_device or connection_refused or happy_eyeballs or ipv6)"
-    - coverage report --show-missing --skip-covered --fail-under 86  # [linux]
+    - coverage report --show-missing --skip-covered --fail-under 85  # [linux]
```

Comment:
☝️ Are you happy with that @bollwyvl?

Reply:
Yep, 1% on a release is no cause for concern... 10% worth looking into!